packages/shared/src/styles/base.css

```diff
@@ -215,6 +218,9 @@ body {
   --theme-gradient-cabbage: theme('colors.cabbage.10')66;
   --theme-gradient-onion: theme('colors.onion.10')66;
 
+  --theme-overlay-cabbage-opacity: theme('colors.cabbage.90')A3;
+  --theme-overlay-onion-opacity: theme('colors.onion.40')A3;
```

Comment:
Are these supposed to switch when in light mode? 90 to 40 then 40 to 90?

Also, I think we need a more specific name for the use case instead of `opacity`. Similar to `theme-hover`, etc.

Reply:
Yeah, talked about naming with Chris, we really need to find something better here but we did make this as a consensus for now. All these generic `opacity` and `theme` names are not really good since they do not say "for what is this style used for" but say this style represents these values".

Reply:
It's because mainly the "what" changes or has multi purpose, for now these are good enough as they once off and needed just for one case.
We can decide on better naming when we get a breather and maybe re-do the whole color naming?